tests/test_gitlab.py

```diff
@@ -129,6 +130,43 @@ def test_to_taskwarrior(self):
             self.arbitrary_extra
         )
 
+        expected_output = {
+            'project': self.arbitrary_extra['project'],
+            'priority': self.service.default_priority,
+            'annotations': [],
+            'tags': [u'feature'],
+            issue.URL: self.arbitrary_extra['issue_url'],
+            issue.REPO: 'project',
+            issue.STATE: self.arbitrary_issue['state'],
+            issue.TYPE: self.arbitrary_extra['type'],
+            issue.TITLE: self.arbitrary_issue['title'],
+            issue.NUMBER: self.arbitrary_issue['iid'],
+            issue.UPDATED_AT: self.arbitrary_updated.replace(microsecond=0),
+            issue.CREATED_AT: self.arbitrary_created.replace(microsecond=0),
+            issue.DUEDATE: self.arbitrary_duedate,
+            issue.DESCRIPTION: self.arbitrary_issue['description'],
+            issue.MILESTONE: self.arbitrary_issue['milestone']['title'],
+            issue.UPVOTES: 0,
+            issue.DOWNVOTES: 0,
+            issue.WORK_IN_PROGRESS: 1,
+            issue.AUTHOR: 'john_smith',
+            issue.ASSIGNEE: 'jack_smith',
+            issue.NAMESPACE: 'arbitrary_namespace',
+            issue.WEIGHT: 3,
+        }
+        actual_output = issue.to_taskwarrior()
+
+        self.assertEqual(actual_output, expected_output)
+
+    def test_work_in_progress(self):
+        arbitrary_issue_2 = self.arbitrary_issue
+        arbitrary_issue_2['work_in_progress'] = 'false'
```

Comment:
Because this line mutates the class attribute `arbitrary_issue`, it will affect subsequent testcases.